row/doc/index.md

```diff
@@ -0,0 +1,71 @@
+```js script
+import '@divriots/dockit-core/row/dockit-row.define.js';
+```
+
+# dockit-row
+
+Component to render children in a row.
+
+## Setup
+
+```js
+import '@divriots/dockit-core/row/dockit-row.define.js';
+```
+
+## Demo
+
+```js preview-story
+import { html } from 'lit';
+import '~/row/dockit-row.define';
+import '~/do-dont/define-all';
```

Comment:
please follow the style of other MDJS stories and use full bare name specifiers, e.g.
and as in other remark - extensions are logical in all code shown to users in docs

```js
import '@divriots/dockit-core/row/dockit-row.define.js';
import '@divriots/dockit-core/do-dont/define-all.js';
```

this is great because user then can see the code exactly like they will have it in their projects and can just copy-paste it, it's general practice for doc sites to show code like that

Reply:
@mveres this wasn't fixed everywhere, can you please have a look at all stories code in docs?
e.g. `dockit-columns` still uses `~/`, I didn't check all, hope you can do it

Reply:
yes, thank you